test/ui-test/testSuites/suite_status/shared/scripts/sections/wallet_names.py

```diff
@@ -62,6 +62,7 @@
 mainWindow_Add_Account_Popup_Seed_Phrase_12 = {"container": mainWallet_Add_Account_Popup_Main, "type": "StatusBaseText", "objectName": "seedPhraseInputPlaceholder11", "visible": True}
 
 mainWallet_Add_Account_Popup_Footer = {"container": statusDesktop_mainWindow, "type": "StatusModalFooter", "unnamed": 1, "visible": True}
+mainWallet_Authenticate_Popup_Footer_Add_Account = {"container": mainWallet_Add_Account_Popup_Footer, "text": "Authenticate", "type": "StatusBaseText", "unnamed": 1, "visible": True}
```

Comment:
Is this really necessary? If so, we need to define it differently because you're using a property that can easily change (the text will change if the app is in another language, for example). It must be defined with an `objectName`.

Reply:
@noeliaSD I was thinking about it, but opted out to keep the way which was already there. If you take a look at the line below (the line `66`) you will see that those 2 are referring to the same place, same button actually, but depends on the context the text is changed after successful authentication.

Reply:
So, I suggest to change both cases.. adding an `objectName` to define the object correctly and remove dependency between properties like `text`. Using these type of properties make the tests weaker. I know we still have some object definitions with these dependences but if we catch them, we should try to refactor them!

Reply:
Yes, I agree, of course we need to do that, but since there are much more than just a few places where we're referring to text that/similar way, would be good to have a ticket for it and refactor them all across all tests in a single go, otherwise will always be a chance to propagate the issue.

Reply:
Well, the refactor can be done in another ticket but we should not add new issues! :)